app/views/request/show.html.erb

```diff
@@ -165,6 +165,17 @@
     </div>
   <% end %>
 
+  <% unless @render_to_file %>
```

Comment:
Please ignore the oversized "Help" link next to "Report this request", that's my browser accessibility setting stepping in and making a mess!